clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1424,11 +1434,10 @@ template <typename T> class trace_converter_t {
                     // not-inside-a-block main loop.
                 }
             } else {
-                // Other than kernel event markers checked above, markers should be
-                // only at block boundaries, as we cannot figure out where they should go
-                // (and could easily insert them in the middle of this block instead
-                // of between this and the next block, with implicit instructions added).
-                // Thus, we do not append any other markers.
+                // All markers should be only at block boundaries, as we cannot
```

Comment:
This is not accurate: kernel markers can be intra-block.  The original comment should probably be left alone.

Reply:
Done.